_posts/2023-04-16-18f-checks-in-with-the-dawson-project-at-the-us-tax-court.md

```diff
@@ -19,9 +18,13 @@ excerpt: >
   The United States Tax Court is an independent federal court that settles disputes between taxpayers and the Internal Revenue Service. Since the 1980s, the Court has used a modified commercial software product to manage its operations. Last year the Court introduced DAWSON, a modern open-source case management system developed with assistance from an industry partner and 18F. DAWSON streamlined Court operations and changed the way the public interacts with the Court.
 ---
 
+_This is the third in a series of blog posts where 18F checks in with our partners throughout the government. Previously in this series: [18F Checks In With Emily Read and the USGS Water Resources Mission Area Projects](https://18f.gsa.gov/2023/02/24/18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects/)._
+
 ![The USTC DAWSON electronic case file system's homepage]({{ site.baseurl }}/assets/blog/ustc-dawson-checkin/dawson-home.png)
+The current home page for the US Tax Court's electronic case management system
+{: .font-alt-xs }
```

Comment:
Minor update to add a caption.

Reply:
Oooh, good catch! I'd add a period at the end of the caption sentence.

Reply:
Also "U.S." as opposed to "US": https://content-guide.18f.gov/our-style/specific-words-and-phrases/

Looks like we should make that change to the title of the blog post as well! 😄

Reply:
Hmmmm, I'm not quite sure! Maybe either way is fine? @LizCamerlin, any thoughts/opinions on this one?

Reply:
This has led me down a hole of good writing advice, like the dos and don'ts for captions here: https://getitwrite.ca/2016/03/09/caption-dos-and-donts/ 

Now I'm thinking I should add more to this caption to make it a little more interesting/give a preview of the contents of the post. There's probably a lot of ways it could be punched up, but I'll try to think of some and seek out opinions soon.

Reply:
What do people think of the updated caption? @alexsobledotgov @LizCamerlin @mark-meyer

Reply:
I have an alternative caption idea here:

> This is the current home page for DAWSON. The Tax Court’s electronic filing and case management system, launched in 2020, is named for the Court’s longest-serving judge, Howard A. Dawson, Jr. This system is entirely web-based and open source.

Reply:
Let me know if you have a preference!

Reply:
@julialeague I love the updated caption Julia — it reads really well and tells a concise story! Thank you for consulting best practices guidance on this one!

Reply:
> @julialeague I love the updated caption Julia — it reads really well and tells a concise story! Thank you for consulting best practices guidance on this one!

Looks great- @julialeague and @alexsobledotgov